### ISSUE TITLE
Error logging in

### ISSUE THREAD

**Issue by darkestmoders** [poky1084/HiloBot-4.0#6]
Status: Error Logging in. (Code: Forbidden)
When trying to login with API
Probably stake updated their API location, please fix/upgrade this bot
thanks a lot

**Comment by poky1084** [comment]
Yes, looks they added another layers of protection to their site, I updated the bot now you shall try this one
